### ISSUE TITLE
Integration of resequenced samples strategy

### ISSUE THREAD

**Issue by gilstel** [satijalab/seurat#5755]
We experienced problems with clustering a cancer dataset whereby a few cells from a big cluster labeled "cancer" contaminate additional clusters.

![4t1pcs30res0_4namedCellTypeUmapSctPcs30Res0_4ClrByClust](https://user-images.githubusercontent.com/50887514/158956097-cd6119ce-4f8c-4a29-8482-8457b786d28c.jpg)

Experiment desription -
The same (4 samples) 10x Genomics libraries were sequenced on two different flowcells in parallel. Same samples were merged using cell ranger and the web report showed that the saturation of several samples was low (around 0.3). Therefore all samples were resequenced after 3 months (due to various constraints). Same samples were again merged using cell ranger and the saturation level increased for most samples.

If we were to treat samples that were run on different flowcells as different sample we would have 12 (4 samples x 3 flowcells).
During the Seurat integration process, the same cells appearing in different samples were named differently and there is an illusion that we have more cells than originally processed. Is this strategy recommended or should we try to integrate the data after it is merged by cell ranger? I should also add that BCR and TCR libraries from the same cells were also sequenced and cell ranger takes into account BCR, TCR and GEX information when merging all data or deciding to throw away cells \ data.

Basically I am asking what is the best option for integration -

1. Treat each sample in each flowcell separately and integrate 12 samples

1. Merge data for same samples using cell ranger and then integrate the 4 samples

Your input would be much appreciated

**Comment by torkencz** [comment]
Hi. I would probably choose your second strategy as you are sequencing same cells over again.
